atom/common/api/atom_api_native_image.cc

```diff
@@ -607,18 +607,26 @@ bool Converter<mate::Handle<atom::api::NativeImage>>::FromV8(
 
 namespace {
 
+using atom::api::NativeImage;
```

Comment:
slight preference towards minimizing diff by making this change in a later PR, as it's not necessary for the change this PR is purportedly making.

Reply:
good point, but in this case I had to re-arrange the code anyway to expose both the class and the API, so it would not make the change much smaller